hammerblade/torch/kernel/hb_tiled_for.hpp

```diff
@@ -233,22 +330,22 @@ inline void hb_tiled_foreach_conversion(HBTensor<scalar_dst> res,
     if (end - start > 4) {
       for (; idx < end - 4; idx += 4) {
         scalar_src input_dp_0 = *(input_data);
-        __remote scalar_dst* res_dp_0 = (res_data);
+        scalar_dst* res_dp_0 = (res_data);
```

Comment:
So here I just want to make sure we are not just reverting what @vb000 have. And if we do, we need good reasons.

Are those `remote` keywords removed because of conflict resolving or because we are trying to put functor at the end?